hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/HederaEvmWorldState.java

```diff
@@ -109,6 +109,11 @@ protected Updater(
             this.evmProperties = evmProperties;
         }
 
+        @Override
+        public Address newContractAddress(Address sponsor) {
+            return null;
```

Comment:
We can create a new interface, that generates the next possible Address, similar to [here](https://github.com/hashgraph/hedera-services/blob/bfd8d9bbb9720fd6da002a2f4c1d55593c75cf1e/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/HederaWorldState.java#L146). Then on mirror-node side we can implement the interface and use the latest contract id to calculate the next possible one.

In hedera for the AccountID/ContractID/TokenID, etc. we use a special sequencer that incrementally generate the next possible ID value. On mirror-node side, however, we should rely on the latest saved ID in the Postgre DB.